simulator/scheduler/config/wasm.go

```diff
@@ -0,0 +1,71 @@
+package config
+
+import (
+	"golang.org/x/exp/maps"
+	"golang.org/x/xerrors"
+	"k8s.io/apimachinery/pkg/util/sets"
+	configv1 "k8s.io/kube-scheduler/config/v1"
+	"k8s.io/kubernetes/pkg/scheduler/apis/config"
+	"k8s.io/kubernetes/pkg/scheduler/apis/config/scheme"
+	"k8s.io/kubernetes/pkg/scheduler/framework/runtime"
+	wasm "sigs.k8s.io/kube-scheduler-wasm-extension/scheduler/plugin"
+)
+
+var wasmRegistry runtime.Registry
```

Comment:
I don't see the need of having a new variable to store the registries for wasm plugins.
Like what you did previously, can we just use the existing `outOfTreeRegistries`?

I replied the last your comment.
https://github.com/kubernetes-sigs/kube-scheduler-simulator/pull/393#discussion_r1838653123